API/SearchUser.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
I think we don't need to make this a public endpoint (we discussed this during our meeting in the library and I had a hard time articulating why, but basically this should be an internal backend thing and not exposed to the frontend). If you want this to be used in your other endpoints, you can just make a function that does all this and include it in your other files that use this endpoint. If I'm misunderstanding def lmk